scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -118,6 +116,41 @@ public static BuildTargetResult PublishDebFileToDebianRepo(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target]
+        [Environment("DOCKER_HUB_REPO")]
```

Comment:
I just read your PR message. If you are targeting only ubuntu, please add this attribute - https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dotnet-cli-build/PublishTargets.cs#L106